tests/h/activity/query_test.py

```diff
@@ -387,12 +387,8 @@ def test_it_returns_each_annotation_presented(self, annotations, pyramid_request
             for bucket in timeframe.document_buckets.values():
                 presented_annotations.extend(bucket.presented_annotations)
 
-        for annotation in annotations:
-            for presented_annotation in presented_annotations:
-                if presented_annotation["annotation"].annotation == annotation:
-                    break
-            else:
-                assert False
+        for presented_annotation, annotation in zip(presented_annotations, annotations):
+            assert presented_annotation["annotation"].annotation == annotation
```

Comment:
These objects just "happen" to line up exactly with a zip. I don't know if this is a feature of the code or an accident, but we can use it to skip the conditionality here.

My intuition is this is safe to do. If not we can revisit if it ever starts causing trouble.